spk/syncthing-inotify/Makefile

```diff
@@ -0,0 +1,33 @@
+SPK_NAME = syncthing-inotify
+SPK_VERS = 0.8.4
+SPK_REV = 1
+SPK_ICON = src/${SPK_NAME}.png
+BETA = 1
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
+
+DEPENDS = cross/$(SPK_NAME)
+SPK_DEPENDS = syncthing
```

Comment:
Should we set a minimum Syncthing package version here? I would probably feel more comfortable in setting the new Syncthing version here, to make sure people are on the latest package version before installing inotify.
For the formatting, check e.g. packages that rely on the Python package(s).